alert-common/src/main/java/com/synopsys/integration/alert/common/security/EncryptionUtility.java

```diff
@@ -43,7 +43,7 @@
     private static final Logger logger = LoggerFactory.getLogger(EncryptionUtility.class);
     private static final String DATA_FILE_NAME = "alert_encryption_data.json";
     private static final String SECRETS_ENCRYPTION_PASSWORD = "ALERT_ENCRYPTION_PASSWORD";
-    private static final String SECRETS_ENCRYPTION_SALT = "ALERT_ENCRYPTION_SALT";
+    private static final String SECRETS_ENCRYPTION_SALT = "ALERT_ENCRYPTION_GLOBAL_SALT";
```

Comment:
Does this affect the environment variable name?

Reply:
No it does not.  We are using the wrong variable name for the secret.